src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/endpoints/AbstractBitbucketEndpoint/config.jelly

```diff
@@ -6,5 +6,6 @@
     <f:entry field="credentialsId" title="${%Credentials}">
       <c:select context="${app}"/>
     </f:entry>
+    <st:include class="${descriptor.clazz}" page="manage-hooks-detail.jelly" optional="true"/>
```

Comment:
Please use a `doFillItems` construction in descriptor instead. This is completely over kill creating a separate jelly file.

Reply:
Will do. To be honest, this was the first solution that I found and I just picked it. No experience with Jelly whatsoever 😆